### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by jaferIdrees**
### Project requirements

- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, created a React component that matches the required design:

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+class Calculator extends React.Component {
+  keys = ['AC', '+/-', '%', '÷',
+    7, 8, 9, 'x',
+    4, 5, 6, '-',
+    1, 2, 3, '+',
+    0, '.', '='];
+
+  keysElements = this.keys.map((key) => (<CalculatorKey value={key} key={key.toString()} />));
+
```
Comment (BenMukebo) [inline src/components/Calculator.js]: - [ ] Great job so far 👏🏻 But, According to the project requirements, The Calculator.js component should be a class-based component and not a stateless function component. Your code might look like this 👇🏻 


```javascript
// ./src/components/Calculator.js

import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator;
```

- Kindly make this 👍🏻

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,8 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<Calculator />);
 }
```
Comment (BenMukebo) [inline src/App.js]: - [ ] Same here, According to the project requirements, The `App.js` component should be a class-based component and not a stateless function component. Your code might look like this 👇🏻 

**./src/App.js**

```javascript
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

- Kindly make this 👍🏻

File: README.md [hunk 3]
```diff
@@ -1,3 +1,48 @@
+## Math Magicians
+
+## Built With
```
Comment (BenMukebo) [inline README.md]: - [ ]  Please, add a meaningful  `Project description. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯

File: README.md [hunk 4]
```diff
@@ -1,3 +1,48 @@
...
+- Twitter: [@twitterhandle](https://twitter.com/jafel_l)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jaferll)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```
Comment (BenMukebo) [inline README.md]: - [ ] When I click the `issues page` link that you added, I get this 404 error, Kindly make sure you correctly add the issues page link , current the link is broken

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @jaferIdrees,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Descriptive PR 👍🏻 
- No linter errors. ✔️ 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @jaferIdrees, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**